EmployeeDepartment/src/main/java/com/example/EmployeeDepartment/services/EmployeeServiceImpl.java

```
@@ -0,0 +1,69 @@
+package com.example.EmployeeDepartment.services;
+
+import com.example.EmployeeDepartment.DAO.DepartmentRepository;
+import com.example.EmployeeDepartment.DAO.EmployeeRepository;
+import com.example.EmployeeDepartment.entity.Department;
+import com.example.EmployeeDepartment.entity.Employee;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class EmployeeServiceImpl implements EmployeeService {
+    @Autowired
+    private EmployeeRepository employeeRepository;
+    @Autowired
+    private DepartmentRepository departmentRepository;
+    @Override
+    public List<Employee> getEmployees() {
+        return employeeRepository.findAll();
+    }
+
+    @Override
+    public Employee addEmployee(Employee employee) {
+        //get department object if present
```

if already exists should throw an error